tests/baselines/reference/templateStringInModuleName.errors.txt

```diff
@@ -1,32 +1,32 @@
+tests/cases/conformance/es6/templates/templateStringInModuleName.ts(1,1): error TS1433: Unexpected keyword or identifier.
 tests/cases/conformance/es6/templates/templateStringInModuleName.ts(1,1): error TS2304: Cannot find name 'declare'.
-tests/cases/conformance/es6/templates/templateStringInModuleName.ts(1,9): error TS1005: ';' expected.
 tests/cases/conformance/es6/templates/templateStringInModuleName.ts(1,9): error TS2580: Cannot find name 'module'. Do you need to install type definitions for node? Try `npm i --save-dev @types/node`.
-tests/cases/conformance/es6/templates/templateStringInModuleName.ts(1,21): error TS1005: ';' expected.
+tests/cases/conformance/es6/templates/templateStringInModuleName.ts(1,16): error TS1442: Template literal not allowed as a string at this position.
+tests/cases/conformance/es6/templates/templateStringInModuleName.ts(4,1): error TS1433: Unexpected keyword or identifier.
 tests/cases/conformance/es6/templates/templateStringInModuleName.ts(4,1): error TS2304: Cannot find name 'declare'.
-tests/cases/conformance/es6/templates/templateStringInModuleName.ts(4,9): error TS1005: ';' expected.
 tests/cases/conformance/es6/templates/templateStringInModuleName.ts(4,9): error TS2580: Cannot find name 'module'. Do you need to install type definitions for node? Try `npm i --save-dev @types/node`.
-tests/cases/conformance/es6/templates/templateStringInModuleName.ts(4,24): error TS1005: ';' expected.
+tests/cases/conformance/es6/templates/templateStringInModuleName.ts(4,16): error TS1442: Template literal not allowed as a string at this position.
 
 
 ==== tests/cases/conformance/es6/templates/templateStringInModuleName.ts (8 errors) ====
     declare module `M1` {
     ~~~~~~~
+!!! error TS1433: Unexpected keyword or identifier.
+    ~~~~~~~
 !!! error TS2304: Cannot find name 'declare'.
             ~~~~~~
-!!! error TS1005: ';' expected.
-            ~~~~~~
 !!! error TS2580: Cannot find name 'module'. Do you need to install type definitions for node? Try `npm i --save-dev @types/node`.
-                        ~
-!!! error TS1005: ';' expected.
+                   ~~~~
+!!! error TS1442: Template literal not allowed as a string at this position.

```

Comment:
The wording of this doesn’t help me much. Do we know enough to say that single/double-quoted strings are valid wherever we give this error?

Reply:
Sure, we can check `expression.tag.escapedText`. I'm having a tough time finding test cases other than that... I'll come back to this soon after trying to ruminate on it.

Reply:
Ok I switched it to the more specific `Module declaration names may only use ' or \" quoted strings.` _(side note: I couldn't find any reference for referring to non template literal strings; is that the right nomenclature?)_. In theory we could only use the module-specific diagnostic if the `expression.tag.escapedText` is `"module"` but I couldn't get any test cases to fire with any other text.

This worries me a little bit because my instinct says this might give a module-specific diagnostic in an edge case not yet tested for... but 🤷 .